<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDialog</class>
 <widget class="QDialog" name="PasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password required</string>
  </property>
  <property name="toolTip">
   <string>Should you forget this word, there is no easy way to recover any of your content</string>
  </property>
  <widget class="QLabel" name="passwordPromptLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter password for protection of your messages</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>361</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>If you forget this word, there is no simple way to recover your contents</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>109</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="echoPwdCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Show password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
